src/components/SignIn/SignIn.test.js

```diff
@@ -41,10 +44,10 @@ describe('SignIn', () => {
   it('should show error on response failure', async () => {
     const mockedRequest = mockSignInFailure(fakeCredentials);
 
-    const { getByTestId, queryByText } = render(<SignIn />);
+    const { getByLabelText, getByTestId, queryByText } = render(<SignIn />);
     const submitButton = getByTestId('submit-button');
```

Comment:
Why aren't you changing also the buttons?

Reply:
Maybe I should. If i were to do that I would think of using the getByText selector. One problem with that approach is that for example on the sign in there are more than one sign in text. So it does not know which one to take.

One posible solution would be to change either of the texts to something else. What do you guys think?

Reply:
There are two bottoms with `Sign In` as the text? Or you can't query by component, just the text?

Reply:
You can only query by text, so this would bring you two results, the button and the title element.

Reply:
same thing here, you can use getByRole for this https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-the-wrong-query

Reply:
The get by role query solves this problem.